src/catalog/product-create-bundle.md

```diff
@@ -170,6 +170,18 @@ _Choose Template_
     _Item Detail_
 
 1. Repeat these steps for each item you want to add to the bundle.
+   
+1. To change the order of items in a bundle section, click the **Move** ( ![Move icon]({% link images/images/btn-drag.png %}){: .Inline} ) icon at the beginning of the row, and drag the item to the new position. 
```

Comment:
```suggestion
1. To change the order of items in a bundle section, click the **Move** ( ![Move icon]({% link images/images/btn-drag.png %}){: .Inline} ) icon at the beginning of the row and drag the item to the desired ordered position. 
```

Reply:
I think this is OK as is.

Reply:
Can't quite bring myself to use the word "desired" in user docs!  (too much longing and craving there!)  I think "drag the item into position" says it all! :-)